gcp/basic-gke-cluster/README.md

```diff
@@ -47,51 +47,51 @@ To run the YAML that this blueprint generates, you need:
 
 ## Information required
 
-This blueprint requires:
+This blueprint requires a:
 
-* GCP project ID (see section below for instructions)
-* A GCP region
+* GCP project ID
+* GCP region
 
-### Creating Google Cloud project
+### Create a Google Cloud project
 
 #### Authenticate to gcloud
-Before configuring gcloud CLI you can check available Zones and Regions nearest to your location
+Before configuring gcloud CLI you can check for available Zones and Regions nearest to your location:
 
 ```sh
 gcloud compute regions list
 
 gcloud compute zones list
 ```
 
-Follow gcloud init and select default Zone Ex. europe-west1, make sure you use the same zone as the one selected during blueprint execution.
+Follow `gcloud init` and select default Zone (for example, `europe-west1`). Make sure you use the same zone as the one selected during blueprint execution.
 
 ```sh
 gcloud init
 ```
 
-> NOTE: You need to have proper permissions and privileges in the GCP account to execute these commands. If you are using a personal account you should be having these as you will be the admin. If you are using a company/enterprise account please check with your account administrator. 
+> Note: You must have the appropriate permissions in the GCP account to execute these commands. If you are using a personal account, you should have these permissions, as you are the admin. If you are using a company/enterprise account, check with your account administrator.
 
 #### Set up environment
 
-> NOTE: If you are creating the Project via GUI instead of below commands, there will be a project number, a project name and a project ID when you initialize the project, and only ID should be exported as `TF_ADMIN` variable.
+> Note: If you are creating the Project using the GUI instead of the commands below, there will be a project number, a project name and a project ID when you initialize the project, and only ID should be exported as `TF_ADMIN` variable.
```

Comment:
If Project is supposed to be capitalized, it should be capitalized everywhere. IMO, it should not be capitalized (along the lines of "zones" and "regions" above)

Reply:
yup